packages/lab/src/calendar/internal/CalendarNavigation.tsx

```diff
@@ -240,9 +253,15 @@ export const CalendarNavigation = forwardRef<
         onSelectionChange={handleMonthSelect}
         {...MonthDropdownProps}
       >
-        {months.map((month) =>
-          OptionWithTooltip(month, formatMonth, isOutsideAllowedMonths(month))
-        )}
+        {months.map((month) => (
+          <OptionWithTooltip
+            key={formatMonth(month)}
+            item={month}
+            formatDate={formatMonth}
+            disabled={isOutsideAllowedMonths(month)}
+            tooltipContent="This month is out of range"
```

Comment:
There's some i18n to be solved with fixed strings

Reply:
I'll make a note of this in the issue, we have fixed strings in a couple of places around calendar, so needs a solution that will address all of them